travel/admin.py

```diff
@@ -2,10 +2,17 @@
 
 # Register your models here.
 
-from .models import Event, Location, Principal, Travel, EventLocationPrincipalTravel
+from .models import Event, Location, Principal, Travel, EventLocationPrincipalTravel, Trip, TripEvent, Region, EventType, Agency, Office, CountryRegion
 
 admin.site.register(Event)
 admin.site.register(Location)
 admin.site.register(Principal)
 admin.site.register(Travel)
 admin.site.register(EventLocationPrincipalTravel)
```

Comment:
Is there a reason to leave the admin in for these, or should we disable from the Admin as a half-measure to show that those models are deprecated?